source/extensions/common/wasm/wasm.cc

```diff
@@ -635,6 +707,25 @@ bool Context::setSharedData(absl::string_view key, absl::string_view value, uint
   return global_shared_data.set(wasm_->id(), key, value, cas);
 }
 
+// Shared Queue
+
+void Context::registerSharedQueue(absl::string_view queue_name) {
+  auto root_context_id = id(); // Assime that Context is a root context.
+  if (root_context_id_ != 0) { // Root contexts have root_context_id_ == 0.
+    root_context_id = root_context_id_;
```

Comment:
Can we add some comment here such as "convert context of queue registration call from stream context to its root context". It is not obvious on what this two lines do.

Reply:
Done.